### DISCUSSION TITLE
BITR difference Gopro 11 and 12

### DISCUSSION THREAD

**Discussion by lyricsdude** [gopro/labs#698] [General]
If I set BITR to 150 on both the Gopro 11 and 12, will both cameras act the same?

Because I read that the Gopro 12 has highly variable bitrates depending on the scene.

**Comment by dnewman-gpsw** [comment]
They will act the same.  Using the BITR extension bypasses the HERO12's variable bitrate engine.

**Reply by lyricsdude** [reply]
thank you :)
